project/movieswyper/models/genre.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+class Genre(models.Model):
+    # From TMDB
+    name = models.TextField(max_length=None)
```

Comment:
We talked about adding the variable "slug" to this but I'm not sure anymore why. Is it still necessary?